mne/time_frequency/tfr.py

```diff
@@ -580,8 +580,8 @@ def __init__(self, info, data, times, freqs, nave, comment=None,
             raise ValueError("Number of times and data size don't match"
                              " (%d != %d)." % (n_times, len(times)))
         self.data = data
-        self.times = times
-        self.freqs = freqs
+        self.times = np.asarray(times)
+        self.freqs = np.asarray(freqs)
```

Comment:
I changed this as if you give a the freqs as a list, it throws a weird error whenever you try to do something with it. But now I realize this is also an API change. I'm not sure if this can break anything though. I assume users usually pass the freqs as an np.array anyway.